sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -111,10 +115,13 @@ export class DisbursementOverawardService {
           application: { applicationNumber: true },
           triggerType: true,
         },
+        addedBy: { firstName: true, lastName: true },
```

Comment:
shouldn't we select `addedDate` ? and do we still need to select  `creator` and  `createdAt`?
![image](https://user-images.githubusercontent.com/77353155/223851266-7e8a5e9c-021f-40f3-95b0-d895655f226d.png).

Reply:
`addedDate: true,` is in the next line.

I think we can remove creator and createdAt now. Thanks!